src/Clock.tsx

```diff
@@ -0,0 +1,61 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+type State = {
+  today: Date,
+};
+
+function correctTime(time: Date) {
+  return time.toUTCString().slice(-12, -4);
+}
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      window.console.info(correctTime(this.state.today));
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    const { clockName } = this.props;
+
+    if (clockName !== prevProps.clockName) {
+      window.console.debug(
+        `Renamed from ${prevProps.clockName} to ${clockName}`,
+      );
+    }
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.timerId);
+  }
+
+  render() {
+    const { clockName } = this.props;
+
+    return (
+      <div className="Clock">
+        <strong className="Clock__name">
+          {clockName}
+        </strong>
+
+        {' time is '}
+
```

Comment:
You can wrap in by span too
```suggestion
        <span> time is </span>
```